src/arrayMethodSort.js

```diff
@@ -4,8 +4,26 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  const defaultSort = (a, b) => (a.toString() > b.toString() ? 1 : -1);
+
+  [].__proto__.sort2 = function(callback = defaultSort) {
+    let isReplaced = true;
+
+    while (isReplaced !== false) {
+      isReplaced = false;
+
+      for (let i = 0; i < this.length - 1; i++) {
+        if (callback(this[i], this[i + 1]) > 0) {
+          const bufferValue = this[i];
+
+          this[i] = this[i + 1];
+          this[i + 1] = bufferValue;
```

Comment:
good job!!

You can use only one line statement
[this[i], this[i + 1]] = [this[i + 1], this[i]];
instead